src/sage/plot/plot3d/parametric_plot3d.py

```diff
@@ -989,7 +989,7 @@ def g(x, y): return x, y+sin(y), x**2 + y**2
     if isinstance(f, Vector):
         f = tuple(f)
 
-    if isinstance(f, (list, tuple)) and len(f) > 0 and isinstance(f[0], (list, tuple)):
+    if isinstance(f, (list, tuple)) and f and isinstance(f[0], (list, tuple)):
         return sum([parametric_plot3d(v, urange, vrange, plot_points=plot_points, **kwds) for v in f])
```

Comment:
may be you can change to `sum(parametric_plot3d(v, urange, vrange, plot_points=plot_points, **kwds) for v in f)` on the way ?